requirements.txt

```diff
@@ -13,7 +13,7 @@ defusedxml==0.7.1
 expandvars==0.9.0
 fastapi[all]==0.89.1
 fastapi-pagination[sqlalchemy]==0.11.4
-fideslang==3.0.0
+fideslang==3.0.1a3
```

Comment:
Wow haven't seen this updated in a while!

Reply:
i know! this'll need to change before we merge but after merging/release fideslang...